govstack_test_harness_api/building_blocks/bb-digital-registries/services/services.py

```diff
@@ -0,0 +1,96 @@
+import os
+from types import SimpleNamespace
+from django.contrib.auth.models import AnonymousUser
+from graphene import Schema
+from graphene.test import Client
+from unittest import mock
+from core.schema import Query as CoreQuery, Mutation as CoreMutation
+from django.apps import apps
+
+app_config = apps.get_app_config('govstack_test_harness_api')
+
+
+def get_client(schema, query, mutation):
+    return Client(schema=schema(query=query, mutation=mutation))
+
+
+def create_base_context():
+    user = mock.Mock(is_anonymous=False)
+    user.has_perm = mock.MagicMock(return_value=False)
+    return SimpleNamespace(user=user)
+
+
+def get_context(request):
+    if request.user.is_authenticated:
+        context = create_base_context()
+        context.user = request.user
+    else:
+        context = SimpleNamespace(user=request.user)
+    return context
+
+
+def get_query_content_values(query_content: dict, registry_name: str, version: str) -> dict:
+    content_values = {}
+    if not query_content:
+        return {}
+
+    json_mapping = app_config.digital_registry.get(registry_name, {}).get(version, {})
+    for input_key, output_key in json_mapping.items():
+        content_values[output_key] = query_content.get(input_key, "")
+    return content_values
+
+
+def get_values_for_insurees(content_values: dict, registry_name: str, version: str) -> dict:
+    json_mapping = app_config.digital_registry.get(registry_name, {}).get(version, {})
```

Comment:
Same comment applies for other uses of the nested values from digital_registry config. Just use execption. Also if you're using same rather complex get it would be good to wrap it in separate function (could be a property in mapping class). This will result in making changes only in one place instead of across the several functions (in spirit of [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle).